functions/src/sections/Section.model.ts

```diff
@@ -3,7 +3,22 @@ import {
   DetailedClassInformation,
 } from '@vikelabs/uvic-course-scraper/dist/types';
 
-export type Section = ClassScheduleListing;
+export type MeetingTimes = {
+  type: string;
+  time: string;
+  days: string;
+  where: string;
+  dateRange: string;
+  scheduleType: string;
+  instructors: string[];
+  building?: string;
+  buildingAccronym?: string;
```

Comment:
```suggestion
  buildingAbbreviation?: string;
```
 I'm not sure `acronym` is the best here given in some cases it's just a shortened version.